source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -331,6 +343,50 @@ TEST_F(NiDCPowerDriverApiTest, SetAttributeViInt64_GetAttributeViInt64ReturnsSam
   EXPECT_EQ(expected_value, get_attribute_value);
 }
 
+TEST_F(NiDCPowerDriverApiTest, ConfigureOutputFunctionAndVoltageLevel_ConfiguresSuccessfully)
+{
+  const char* channel_name = "0";
+  ViReal64 expected_voltage_level = 3.0;
+  ViInt32 expected_output_function_value = dcpower::OutputFunction::OUTPUT_FUNCTION_NIDCPOWER_VAL_DC_VOLTAGE;
+  configure_output_function(channel_name, expected_output_function_value);
+  ::grpc::ClientContext context;
+  dcpower::ConfigureVoltageLevelRequest request;
```

Comment:
Why have you not extracted this into a helper for configure_volatge_level?

Reply:
Yes, looks better now.